vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -137,6 +138,7 @@ def create_weights(
     ):
         del input_size, output_size
         output_size_per_partition = sum(output_partition_sizes)
+        weight_loader = extra_weight_attrs.get("weight_loader")
```

Comment:
Are there cases where we aren't guaranteed to have a weight_loader here?

Reply:
I dont think so - unless we made changes to the `LinearBase` lifecycle